lib/yi_appium_caps_util.rb

```diff
@@ -24,6 +23,13 @@ def update (caps_file_name = './appium.txt')
       end
     end
 
+    def create (platform_name)
+      File.open("./appium.txt","w") do |f|
+        f.puts("[caps]\napp = \" \"\nautomationName = \"YouiEngine\"\ndeviceName = \" \"\nplatformName = \"#{platform_name}\"\nyouiEngineAppAddress = \" \"\n")
```

Comment:
No I call `update()` right after it in the next line which will update the file based on the PlatformName. There is no handling for iOS<10 vs iOS>10; shall I add this here or create a separate PR for it? 

I could add a check for iOS version in the  `update_ios_caps` function